src/crimson/osd/pg.h

```diff
@@ -362,7 +362,8 @@ class PG : public boost::intrusive_ref_counter<
   void on_active_advmap(const OSDMapRef &osdmap) final {
     // Not needed yet
   }
-  epoch_t oldest_stored_osdmap() final {
+
+  epoch_t max_oldest_stored_osdmap() final {
```

Comment:
I should have commented on this before, but this naming is simply confusing -- max oldest stored osdmap doesn't mean much since there's only a single oldest stored osdmap on a particular osd.  Rename this to cluster_osdmap_trim_lower_bound().